src/app/[locale]/specs/permission/en.mdx

```diff
@@ -0,0 +1,347 @@
+- `action` (string array, optional): defines the set of record operations allowed. Allowed values are `create`, `update`, `delete`. If not defined, all operations are allowed
+
+Examples:
+
+```
+# full permission (create, update, delete) on the Bluesky profile record type
+repo:app.example.profile
+
+{
+  "type": "permission",
+  "resource": "repo",
+  "collection": ["app.example.profile"]
+}
+
+# same as above, with actions being explicit
+repo:app.example.profile?action=create&action=update&action=delete
+
+{
+  "type": "permission",
+  "resource": "repo",
+  "collection": ["app.example.profile"],
+  "action": ["create", "update", "delete"]
+}
+
+# full permissions on all record types
+repo:*
+
+{
+  "type": "permission",
+  "resource": "repo",
+  "collection": ["*"]
+}
+
+# delete only permission on all record types
+repo:*?action=delete
+```
+
+### `rpc`
+
+The ability to make authenticated API requests to remote services. This includes both requesting service auth tokens (JWTs) using the `com.atproto.server.getServiceAuth` endpoint on the PDS, and requests proxied via the PDS.
+
+The permission is parameterized by the remote endpoint (`lxm`, short for "Lexicon Method") and the identity of the remote service (the audience, `aud`). Permissions must be restricted by one or the other: the ability to call any API endpoint on any service can not be declared with a single permission.
+
+When included in a permission set, the audience parameter may be inherited from the overall set.
+
+Parameters:
+
+- `lxm` (string array, required, positional): NSID of API endpoints. Wildcard (`*`) is allowed and gives access to all endpoints. Partial wildcards are not supported (eg, can not use `com.example.*`).
+- `aud` (string, semi-required): audience of API requests, as a DID service reference: DID followed by required service type fragment, eg `did:web:api.example.com#srvtype`). Supports wildcard (`*`). Required in granular string representation, and required in lexicon JSON unless `inheritAud` is true.
+- `inheritAud` (boolean, optional): if true, `aud` does not need to be defined.
+
+Examples:
+
+```
+# Submit moderation reports to any service
+rpc:app.example.moderation.createReport?aud=*
+
+{
+  "type": "permission",
+  "resource": "rpc",
+  "lxm": ["app.example.moderation.createReport"],
+  "aud": ["*"]
+}
+
+# Call any RPC method on a specific service
+rpc?lxm=*&aud=did:web:api.example.com%23svc_appview
+
+{
+  "type": "permission",
+  "resource": "rpc",
+  "lxm": ["*"],
+  "aud": ["did:web:api.example.com#svc_appview"]
+}
+```
+
+## `blob`
+
+Ability to upload media files (blobs) to PDS.
+
+Parameters:
+
+- `accept` (array of strings, required): MIME types or partial MIME type glob patterns (`*/*` or `text/*` for example). Same syntax as the `accept` field in the `blob` lexicon type.
```

Comment:
note: Should we add a `collections` field here to restrict which collection can be referencing new blobs ?

Example use case would be "allow app X to upload images for your profile; but not for posts"

Reply:
this isn't implemented anywhere right? I think we hold off

Reply:
I agree we shouldn't add functionality right now, let's just keep blobs broad.